linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_accept_source_route.rule

```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 prodtype: rhel7,fedora
 
-title: 'Configure Kernel Parameter for Accepting Source-Routed Packets for All Interfaces'
+title: 'Configure Kernel Parameter for Accepting Source-Routed Packets for All Interfaces via IPv4'
```

Comment:
`Configure Kernel Parameter for Accepting Source-Routed Packets for All IPv4 Interfaces`

Reply:
@redhatrises An interface is not "IPv4" or "IPv6" -- it can be both. Settings can be independent of the interface; they're primarily settings for the entire ipv4 stack inside the kernel. Further, one interface may have both ipv4 and ipv6, and different settings for the same value whether it is ipv4 or ipv6.

How about making these "Configure Kernel Parameter for Accepting Source-Routed Packets via IPv4" and the default ones "Configure Kernel **Default** Parameter for Accepting Source-Routed Packets via IPv4"?

Reply:
@cipherboy the issue here is clarity. Are you configuring this with the command IPv4?

Reply:
To me  "Configure Kernel Parameter for Accepting Source-Routed Packets via IPv4" implies that the packets are routed via IPv4, not that the configuration is done via IPv4; would you prefer  "Configure Kernel Parameter for Accepting IPv4 Source-Routed Packets" then?

Reply:
> "Configure Kernel Parameter for Accepting IPv4 Source-Routed Packets"

@cipherboy Perfect.